python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -1279,6 +1503,7 @@ def usages(self):
                     "stopTracking",
                     "offsetAzEl",
                     "offsetRADec",
+                    "poriginOffset",
```

Comment:
Was this intentionally spelled like this or should this be `originOffset`? If intentionally then please consider naming it `pOriginOffset` instead here and everywhere else.

Reply:
uhm.. interesting point... unfortunately, this is the name of the command in the xml and I can't change it.

Reply:
Maybe something for a future ticket?